server/utils/secrets/secret_ref_utils.py

```
@@ -226,20 +228,22 @@ def get_user_token_data(self, user_id: str, provider: str) -> Optional[Dict[str,
+                f"""SELECT secret_ref, client_id, client_secret
                    FROM user_tokens
-                   WHERE (user_id = %s OR org_id = %s)
+                   WHERE user_id = %s
                      AND provider = %s
                      AND secret_ref IS NOT NULL
                      AND is_active = TRUE
+                     {clause}
                    ORDER BY CASE WHEN user_id = %s THEN 0 ELSE 1 END
                    LIMIT 1""",
-                (user_id, org_id, provider, user_id)
+                (user_id, provider_base) + clause_params + (user_id,),
```

Ruff (0.15.12)

[warning] Consider `(user_id, provider_base, *clause_params, user_id)` instead of concatenation

Replace with `(user_id, provider_base, *clause_params, user_id)`

(RUF005)